odoo/addons/base/res/res_company_view.xml

```diff
@@ -6,11 +6,11 @@
             <field name="model">res.company</field>
             <field name="arch" type="xml">
                 <form string="Company">
-                  <sheet>
+                  <sheet id="company_sheet">
```

Comment:
Is it necessary as there should be only one sheet ? Does the xpath have issues finding it without the id ?

Reply:
Up !